src/wp-admin/includes/class-wp-posts-list-table.php

```diff
@@ -1432,47 +1482,53 @@ public function inline_edit() {
 				<span class="input-text-wrap"><input type="text" name="post_name" value="" /></span>
 			</label>
 
-	<?php endif; // $bulk
-	endif; // post_type_supports title ?>
+				<?php
+	endif; // $bulk
+	endif; // post_type_supports title
+			?>
 
-	<?php if ( !$bulk ) : ?>
+			<?php if ( ! $bulk ) : ?>
 			<fieldset class="inline-edit-date">
 			<legend><span class="title"><?php _e( 'Date' ); ?></span></legend>
 				<?php touch_time( 1, 1, 0, 1 ); ?>
 			</fieldset>
 			<br class="clear" />
-	<?php endif; // $bulk
-
-		if ( post_type_supports( $screen->post_type, 'author' ) ) :
-			$authors_dropdown = '';
-
-			if ( current_user_can( $post_type_object->cap->edit_others_posts ) ) :
-				$users_opt = array(
-					'hide_if_only_one_author' => false,
-					'who' => 'authors',
-					'name' => 'post_author',
-					'class'=> 'authors',
-					'multi' => 1,
-					'echo' => 0,
-					'show' => 'display_name_with_login',
-				);
-				if ( $bulk )
-					$users_opt['show_option_none'] = __( '&mdash; No Change &mdash;' );
-
-				if ( $authors = wp_dropdown_users( $users_opt ) ) :
-					$authors_dropdown  = '<label class="inline-edit-author">';
-					$authors_dropdown .= '<span class="title">' . __( 'Author' ) . '</span>';
-					$authors_dropdown .= $authors;
-					$authors_dropdown .= '</label>';
-				endif;
-			endif; // authors
-	?>
-
-	<?php if ( !$bulk ) echo $authors_dropdown;
+				<?php
+	endif; // $bulk
+
+			if ( post_type_supports( $screen->post_type, 'author' ) ) :
+				$authors_dropdown = '';
+
+				if ( current_user_can( $post_type_object->cap->edit_others_posts ) ) :
+					$users_opt = array(
+						'hide_if_only_one_author' => false,
+						'who'                     => 'authors',
+						'name'                    => 'post_author',
+						'class'                   => 'authors',
+						'multi'                   => 1,
+						'echo'                    => 0,
+						'show'                    => 'display_name_with_login',
+					);
+					if ( $bulk ) {
+						$users_opt['show_option_none'] = __( '&mdash; No Change &mdash;' );
+					}
+
+					if ( $authors = wp_dropdown_users( $users_opt ) ) :
+						$authors_dropdown  = '<label class="inline-edit-author">';
+						$authors_dropdown .= '<span class="title">' . __( 'Author' ) . '</span>';
+						$authors_dropdown .= $authors;
+						$authors_dropdown .= '</label>';
+					endif;
+				endif; // authors
+				?>
+
+				<?php
+				if ( ! $bulk ) {
+					echo $authors_dropdown;}
```

Comment:
Ending brace should be on new line.

Reply:
Agreed, will add to the amends already made locally ready to update the PR.